avm/res/sql/server/main.bicep

```
@@ -465,17 +481,28 @@ module server_keys 'key/main.bicep' = [
   }
 ]
 
-module server_encryptionProtector 'encryption-protector/main.bicep' = if (encryptionProtectorObj != null) {
+module cmk_key 'key/main.bicep' = if (customerManagedKey != null) {
```

```suggestion
module server_key 'key/main.bicep' = if (customerManagedKey != null) {
```
Just usually the approach used for naming child-resource deployments. If you want to keep `cmk_key`, feel free to resolve

---

I'd keep it, as we have a `server_keys` (user can manually add  a number of keys), and I believe renaming `cmk_key` to `server_key` could confuse myself in the future when looking at the code